Header.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+#include <iostream>
+#include <string>
+using namespace std;
+class Movie
+{
+public:
+    int id;
+    string title;
+    string ranking;
+    int release_date;
+    int number_of_tickets;
+    int price_of_ticket;
+    string comment;
+    int get_id(){
+        return id;
+    }
+    string get_title(){
+        return title;
+    }
+    string get_ranking(){
+        return ranking;
+    }
+    int get_release_date(){
+        return release_date;
+    }
+    int get_number_of_tickets(){
+        return number_of_tickets;
+    }
+    int get_price_of_ticket(){
+        return price_of_ticket;
+    }
+    string get_comment(){
+        return comment;
+    };
+    Movie() {};
+    enum FilmType
+    {
+        kAction,
+        kAdventure,
+        kComedy,
+        kCrime,
+        kCriminal,
+        kDetective,
+        kDrama,
+        kFamily,
+        kFantasy,
+        kHorrors,
+        kMelodrama,
+        kRomance,
+        kThriller
+    };
+    Movie(int id, string title, string ranking, int release_date, int  number_of_tickets, int price_of_ticket, string comment)
+    {
+        this->id = id;
+        this->title = title;
+        this->ranking = ranking;
+        this->release_date = release_date;
+        this->number_of_tickets = number_of_tickets;
+        this->price_of_ticket = price_of_ticket;
+        this->comment = comment;
+    }
+    void Print()

```

Comment:
since you decide to use snake_case_approach firs, all functions and methods should follow the same approach: get_price_of_ticket